src/ActionMonitor/ActionMonitor.php

```diff
@@ -197,13 +197,9 @@ function monitorActions() {
 		add_action( 'delete_user', [ $this, 'deleteUser' ], 10, 2 );
 
 		// Post meta updates
-		$meta_types = [ 'user', 'post', 'page' ];
-
-		foreach ( $meta_types as $type ) {
-			add_action( "updated_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-			add_action( "added_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-			add_action( "deleted_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-		}
+		add_action( "updated_post_meta", [ $this, 'modifyMeta' ], 100, 3 );
+		add_action( "added_post_meta", [ $this, 'modifyMeta' ], 100, 3 );
```

Comment:
I think this removes page meta updates. I believe there are separate hooks for `post_meta` and `page_meta` and the post one applies to all CPT's but not to pages

Reply:
Well 🤔 maybe I'm wrong actually :p

Reply:
Yeah I'm wrong 😆

Reply:
ok, I'll be revisiting to add meta tracking for other entities and will make sure pages work when I get there.